### PR TITLE
Add new active contributors

### PR DESCRIPTION
**Pull request by byeong-gil**
Add @erlingrj, @Jakio815, @byeong-gil to active contributors.

### CODE REVIEW

**Comment by byeong-gil** [comment]
@erlingrj, @Jakio815 Could you fill in your URL, image, and blurb? If you don't have a permit to edit this branch, you simply give me the information so that I can fill it in.

**Comment by lhstrh** [comment]
Cool! Thanks, @byeong-gil. Could you change the order to be alphabetical by last name? I think country should correspond to the institution, which for Dongha and Hokeun now is ASU.

Could you also move Anirudh to past collaborators? Thanks!

**Comment by lhstrh** [comment]
@arengarajan99, what is your new affiliation?

**Comment by byeong-gil** [comment]
>Could you change the order to be alphabetical by last name? I think country should correspond to the institution, which for Dongha and Hokeun now is ASU.
Could you also move Anirudh to past collaborators? Thanks!

@lhstrh I did it!

File: packages/lingua-franca/src/templates/pages/community.tsx [hunk 1]
```diff
@@ -106,6 +121,13 @@ const past = [
     continentish: "North America",
     blurb: "Applied Math and CS student at UC Berkeley."
   },
+  {
+    name: "Anirudh Rengarajan",
+    image: "https://avatars.githubusercontent.com/u/44007330?v=4",
+    country: "🇺🇸",
+    continentish: "North America",
+    blurb: "M.S. student @ UC Berkeley.",
```
Comment (lhstrh) [inline packages/lingua-franca/src/templates/pages/community.tsx:129]:
```suggestion
    blurb: "Software Engineer at Bloomberg.",
```

File: packages/lingua-franca/src/templates/pages/community.tsx [hunk 2]
```diff
@@ -26,13 +26,35 @@ const active = [
...
+    country: "🇰🇷",
+    continentish: "East Asia",
+    blurb: "Undergraduate student at Hanyang University.",
+  },
+  {
+    name: "Dongha Kim",
+    image: "https://avatars.githubusercontent.com/u/74869052?s=400&v=4",
+    country: "🇺🇸",
+    continentish: "North America",
+    blurb: "Ph.D. student at Arizona State University",
```
Comment (lhstrh) [inline packages/lingua-franca/src/templates/pages/community.tsx:49]:
```suggestion
    blurb: "Ph.D. student at Arizona State University.",
```

File: packages/lingua-franca/src/templates/pages/community.tsx [hunk 3]
```diff
@@ -26,13 +26,35 @@ const active = [
     continentish: "Europe",
     blurb: "Master student at TU Dresden.",
   },
+  {
+    name: "Erling Rennemo Jellum",
+    url: "https://erlingrj.github.io",
+    image: "https://backends.it.ntnu.no/user-profile-service/rest/files/139f11cf-c2bb-365e-bbbb-559165b9cb31",
+    country: "🇳🇴",
+    continentish: "Europe",
+    blurb: "PhD Candidate at Norwegian University of Science and Technology",
```
Comment (lhstrh) [inline packages/lingua-franca/src/templates/pages/community.tsx:35]:
```suggestion
    blurb: "PhD Candidate at Norwegian University of Science and Technology.",
```

**Review by lhstrh — APPROVED** [review]
Looks good! Thanks, @byeong-gil!

**Pull request merged**